keyboards/k_type/readme.md

```diff
@@ -2,12 +2,16 @@
 
 Firmware for custom keyboard PCB with TKL Key Layout
 
-Keyboard Maintainer: [Kaleb Elwert](https://github.com/belak)  
-Hardware Supported: Input Club K-Type   
-Hardware Availability: [Input Club](https://input.club/k-type/), [Massdrop](https://www.massdrop.com/buy/massdrop-x-input-club-k-type-mechanical-keyboard?utm_source=linkshare&referer=WJJG5M)
+* Keyboard Maintainer: [Kaleb Elwert](https://github.com/belak)
+* Hardware Supported: Input Club K-Type
+* Hardware Availability: [Input Club](https://input.club/k-type/), [Massdrop](https://www.massdrop.com/buy/massdrop-x-input-club-k-type-mechanical-keyboard?utm_source=linkshare&referer=WJJG5M)
```

Comment:
```suggestion
* Hardware Availability: [Input Club](https://input.club/k-type/), [Massdrop](https://www.massdrop.com/buy/massdrop-x-input-club-k-type-mechanical-keyboard)
```

Reply:
Huh, I wonder who's been getting referral credits because it wasn't me! Hah.